tools/rna_tools/music/music.xml

```diff
@@ -88,12 +103,19 @@
     <outputs>
         <data name="output1" format="txt" from_work_dir="chr_ids.txt" />
         <data name="output2" format="txt" from_work_dir="*_mapped_reads.txt" />
+		<data format="bed" name="SSERs_report">
+			<discover_datasets pattern="SSERs_*_designation_and_ext__" directory="." visible="true" />
```

Comment:
You could also `mv SSERs*` into it's own subdir and collect all from this directory with the standard regex.